locales/en/plugin__activemq-artemis-self-provisioning-plugin.json

```diff
@@ -31,7 +31,7 @@
   "auto_create_queues": "Auto create queues",
   "auto_delete_queues": "Auto delete queues",
   "delete_modal_instance_title": "Delete instance ?",
-  "delete": "Delete",
-  "cancel": "Cancel",
-  "preconfirm_delete_broker_message": "The 'test-2' instance will be deleted. Applications will no longer have access in this instance."
+  "delete": "<strong>Delete</strong>",
```

Comment:
We don't need <strong> tag for delete and cancel.

Reply:
okay Ajay, i will reomve the tag i just thought to make delete button to looks  bold so inserted the tag...